v2/concordium/service.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+import "v2/concordium/types.proto";
+
+package concordium.v2;
+
+service Queries {
+  // Return a stream of blocks that arrive from the time the query is made onward.
+  // This can be used to listen for incoming blocks.
+  rpc GetBlocks (Empty) returns (stream BlockInfo);
+
+  // Return a stream of blocks that are finalized from the time the query is
+  // made onward. This can be used to listen for newly finalized blocks. Note
+  // that there is no guarantee that blocks will not be skipped if the client is
+  // too slow in processing the stream, however blocks will always be sent by
+  // increasing block height.
+  rpc GetFinalizedBlocks (Empty) returns (stream FinalizedBlockInfo);
+
+  // Retrieve the information about the given account in the given block.
+  rpc GetAccountInfo (AccountInfoRequest) returns (AccountInfo);
+
+  // Retrieve the list of accounts that exist in the given block.
```

Comment:
We could consider making this more precise. Describing that it is all the accounts that exist at the end of this block.